packages/browser-sdk/src/feature/features.ts

```diff
@@ -28,7 +28,14 @@ export type RawFeature = {
 
 const FEATURES_UPDATED_EVENT = "features-updated";
 
-export type RawFeatures = Record<string, RawFeature | undefined>;
+export type FetchedFeatures = Record<string, FetchedFeature | undefined>;
+export type RawFeature = FetchedFeature & {
```

Comment:
can we rename this? confuses me slightly as a `raw` indicates that it's not manipulated internally, whereas overrides is a form of manipulation.

perhaps `LocalFeature` ?

Reply:
this is unfortunately already exported so renaming would be a breaking change. Added a note